### PR TITLE
New release 0.4.51

### PR DESCRIPTION
**Pull request by jaypontes**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
 
-# Unreleased yet
-* Bugfix: Write undeploy versions correctly into manifest repo
-
 # Change Log
 
+## 0.4.51
+* Bugfix: Write undeploy versions correctly into manifest repo [#412](https://github.com/freiheit-com/kuberpult/pull/412)
+* route /v2/home to index.html in build directory [#454](https://github.com/freiheit-com/kuberpult/pull/454)
+* Fix exec planner [#453](https://github.com/freiheit-com/kuberpult/pull/454)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:12]: The PR is wrong here, also don't need to include fix exec planner, this is something internal

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* Add EnvironmentGroup to getOverview endpoint [#449](https://github.com/freiheit-com/kuberpult/pull/449)
+* Backend: Add environmentGroup to envConfig [#447](https://github.com/freiheit-com/kuberpult/pull/447)
+* refactoring: Rename config to ServerConfig [#446](https://github.com/freiheit-com/kuberpult/pull/446)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:15]: This is something internal, no need to put it here

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* refactoring: rename config to RepositoryConfig [#445](https://github.com/freiheit-com/kuberpult/pull/445)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:16]: This is something internal, no need to put it here

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* Removed misleading tooltip about queues in old UI [#443](https://github.com/freiheit-com/kuberpult/pull/443)
+* Add "development2" to test data [#440](https://github.com/freiheit-com/kuberpult/pull/440)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:18]: This is something internal, no need to put it here

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* Add command to run lint fix [#439](https://github.com/freiheit-com/kuberpult/pull/439)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:19]: This is something internal, no need to put it here

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* Improved readme for testdata [#437](https://github.com/freiheit-com/kuberpult/pull/437)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:20]: This is something internal, no need to put it here

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* Added scripts to create test data [#433](https://github.com/freiheit-com/kuberpult/pull/433)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:21]: This is something internal, no need to put it here

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* Add missing url to docker-compose.yml [#434](https://github.com/freiheit-com/kuberpult/pull/434)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:22]: https://github.com/freiheit-com/kuberpult/pull/434

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
...
+* Fix github actions deprecation errors [#430](https://github.com/freiheit-com/kuberpult/pull/430)
```
Comment (mouhsen-ibrahim) [inline CHANGELOG.md:23]: https://github.com/freiheit-com/kuberpult/pull/434

**Review by mouhsen-ibrahim — CHANGES_REQUESTED** [review]
Please only include important changes from a user point of view, no need to include internal changes as well

File: CHANGELOG.md (hunk 1 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 2 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 3 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 4 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 5 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 6 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 7 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 8 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 9 above)
Comment (jaypontes) [reply CHANGELOG.md]: fixed!

**Pull request merged**